### DISCUSSION TITLE
Welcome to oekofen-pellematic-compact Discussions!

### DISCUSSION THREAD

**Discussion by dominikamann** [dominikamann/oekofen-pellematic-compact#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by lucaferrucci** [comment]
hello, good morning, as promised I am sending you the file of my system, it is quite complete as I have 2 heating circuits with mixer and solar thermal, if you need anything else, ask without problems.
I have excellent relations with oekofen italia and they really liked your work
[jasonAll.zip](https://github.com/dominikamann/oekofen-pellematic-compact/files/10497611/jasonAll.zip)

**Reply by dominikamann** [reply]
Hi @lucaferrucci, thank you for providing me the file. I started a new discussion for better documentation. See: https://github.com/dominikamann/oekofen-pellematic-compact/discussions/3

**Comment by heffner99** [comment]
Hi, is this integration only for Ökofen Pellematic Compact or also for other Ökofen boilers, e.g., PES 32?

**Reply by dominikamann** [reply]
Hi @heffner99,
I only own a Okofen Pellematic. For this I developed this integration. Over the time I extended it for more and more systems and setups. So I can not answer your question but if your heater supports the JSON Interface it could work. Try it out. It there are errors then create an issue for it and I might be able to extend the integration again to also support your setup.
